en/flying/flight_reporting.md

```diff
@@ -0,0 +1,16 @@
+# Flight Reporting
```

Comment:
@bkueng So I moved flight reporting under flying and updated it to current QGC. I think this is not a top level topic, and that this is the right place - OK?

One question, should we better highlight on front page "how to raise issues"? We could then link to this document?

Reply:
I'm ok with that.

> One question, should we better highlight on front page "how to raise issues"? We could then link to this document?

Sure.

Reply:
What is our position on that then? Just raise an issue if you see a problem, or do we push them to ask on slack or whatever first for some pre-triaging?

Reply:
If it's a general usability issue, I'd say it should first go through discuss/slack.
If someone has a log with a specific issue, it can go to github. But I'm not quite sure if there's a better way. Sometimes it's just a setup problems (e.g. vibrations), then it does not need to go to github.

Reply:
Thanks very much. I added the note in https://github.com/PX4/px4_user_guide/pull/102